### PR TITLE
Fix typo on exception message

### PR DESCRIPTION
**Pull request by seongahjo**

### CODE REVIEW

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Review by caisq — APPROVED** [review]

File: tensorflow/java/src/main/java/org/tensorflow/Session.java [hunk 1]
```diff
@@ -225,7 +225,7 @@ public Runner addTarget(String operation) {
       public Reference() {
         synchronized (nativeHandleLock) {
           if (nativeHandle == 0) {
-            throw new IllegalStateException("run() called after the Session was close()d");
+            throw new IllegalStateException("run() called after the Session was closed");
```
Comment (asimshankar) [inline tensorflow/java/src/main/java/org/tensorflow/Session.java]: Would it be clearer to say:
"run() cannot be called on the Session after close()"
?

Comment (seongahjo) [inline tensorflow/java/src/main/java/org/tensorflow/Session.java]: Yes, I think it is much clearer than the original message

**Comment by jhseu** [comment]
Jenkins, test this please

**Comment by asimshankar** [comment]
Looks good to me, but I'll be in and out of internet access for the next couple of days, deferring to @jhseu to merge it in.

**Pull request merged**